docs/transport.md

```
@@ -0,0 +1,486 @@
+├── 职责：消息序列化，协议编码，通讯抽象
+├── 隔离：不知道具体业务，不关心物理传输
+└── 接口：Transport 统一接口
+
+平台层 (HTTP/WebUSB/BLE)
+├── 职责：物理通讯实现，平台适配
+├── 隔离：只关心数据传输，不理解消息含义
+└── 接口：相同的传输协议适配
+```
+
+**💡 设计决策的驱动因素：**
+1. **可扩展性**：新增区块链不影响传输层
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)